src/vm/amd64/calldescrworkeramd64.S

```diff
@@ -45,9 +45,8 @@ NESTED_ENTRY CallDescrWorkerInternal, _TEXT, CallDescrWorkerUnwindFrameChainHand
         push_nonvol_reg rbp
         mov     rbp, rsp
         push_nonvol_reg rbx
-        lea     rsp, [rsp - 8]          // ensure proper alignment of the rsp 
-        set_cfa_register rbp, (4*8)
-
+        alloc_stack 8                   // ensure proper alignment of the rsp 
```

Comment:
The CFA register is `RBP` at this point and using the alloc_stack here would incorrectly adjust the offset - the `RBP` doesn't change in the alloc_stack. I think we should keep the original code here. I wonder if you had to make this change to compensate for the removal of the `.cfi_def_cfa_offset 8` in the `LEAF_ENTRY` macro - see my comment on that one.